lib/components/settings/settings_directory_selector.dart

```diff
@@ -114,15 +115,52 @@ class _DirectorySelectorState extends State<DirectorySelector> {
 
     final dir = Directory(directory);
     _directory = directory;
-    _isEmpty = dir.existsSync() ? dir.listSync().isEmpty : true;
+    _isEmpty = (dir.existsSync() ? dir.listSync().isEmpty : true) || true;
```

Comment:
Why was this changed?

Reply:
Good question, I suppose this could be reversed